.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -31,7 +31,7 @@ assignees: ''
  - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
  - [ ] [GIT] Merge master branch into stable, in owncloud/android-library
  - [ ] [GIT] Merge master branch into stable, in owncloud/android
- - [ ] [DOC] Update owncloud.org/download version numbers (notify rocketchat #marketing)
```

Comment:
With this check, I understood to notify marketing, so they can update the version number from https://owncloud.com/mobile-apps/

Reply:
i guess so, as discussed with @mmattel